### DISCUSSION TITLE
About the wechat group of Deepmodeling

### DISCUSSION THREAD

**Discussion by TerryHangLyu** [deepmodeling/deepmd-kit#1489] [Q&A]
Dear administrators of Deepmodeling,
    I am a beginer of deepMD-kit, and I want to join the wechat group of Deepmodeling to ask some questions. I have sent my application to the wechat id: deeppotential, but receive no response. Is there other access to join the group?

Regards,
Hang LYU

**Comment by likefallwind** [comment]
Hi, we checked the wechat, all new users have been put  into the the wechat group.  And we recommend you ask questions in this discussion board~
